.gitignore

```
@@ -1,2 +1,4 @@
 /_*
 *.mo
+
+venv
```

only ignore `venv/` directories? Maybe limited to the project root:
```suggestion
/venv/
```